CMakeLists.txt

```diff
@@ -539,6 +539,7 @@ endif()
 
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
 if(AVIF_BUILD_TESTS)
+    enable_testing()
```

Comment:
What does enable_testing() do? Why didn't we need this before?

Reply:
See https://github.com/AOMediaCodec/libavif/pull/834#discussion_r800521989